src/__tests__/hover.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import userEvent from '..'
+import {setup} from './helpers/utils'
+
+test('hover', async () => {
+  const {element, getEventCalls} = setup(<button />)
+
+  await userEvent.hover(element)
+  expect(getEventCalls()).toMatchInlineSnapshot(`
+    mouseover: Left (0)
+    mouseenter: Left (0)
+    mousemove: Left (0)
```

Comment:
Got the order of these via: https://jsbin.com/sulogop/1/edit?js,output

Reply:
That's strange, I got the enter/over/move order from React, does it fire events in a different order than the DOM itself?

Reply:
It's possible. React does weird things to events, which is why my playground doesn't use react